src/status_im/ui/screens/chat/extensions/views.cljs

```diff
@@ -30,11 +28,13 @@
                                :useNativeDriver true})])))
 
 (views/defview extensions-view []
-  (views/letsubs [bottom-anim-value (anim/create-value panel-height)
+  (views/letsubs [stickers-panel-height [:chats/stickers-panel-height]
+
+                  bottom-anim-value (anim/create-value @stickers-panel-height)
                   alpha-value       (anim/create-value 0)]
     {:component-did-mount #(show-panel-anim bottom-anim-value alpha-value)}
     [react/animated-view {:style {:background-color :white
-                                  :height    panel-height
+                                  :height    stickers-panel-height
```

Comment:
why do we use stickers-panel-height here? probably it should be `chat-panel-height ?`

Reply:
I can only add that the design for all panels: commands, stickers, images, anything else we might have in the future - shares the same height.

Reply:
I will rename into `chat-panel-height` :+1:, initially, it was created only for stickers and now it needs a rename cause it is reused here.